daiquiri/core/renderers/datacite.py

```diff
@@ -110,11 +115,12 @@ def render_person(self, person_type, person, contributor_type=None):
             name = person.get('name')
             first_name = person.get('first_name')
             last_name = person.get('last_name')
+            name_type = person.get('name_type')
 
             if not name:
                 name = '{}, {}'.format(last_name, first_name)
 
-            self.node(person_type + 'Name', {}, name)
+            self.node(person_type + 'Name', {'nameType': name_type}, name)
```

Comment:
Where is `nameType` coming from? Is that just an additional field in the JSON entry for authors, contributors?

If yes then we should check fro missing values:

```python
if name_type is not None:
    self.node(person_type + 'Name', {'nameType': name_type}, name)
else:
    self.node(person_type + 'Name', {}, name)
```

Except there is some kind of validation of the JSON fields before calling the `render_person` method? In which case we need to add a validation for name_type.

Reply:
The `name_type` comes from 
```python
           name_type = person.get('name_type')
```
This returns `None` if `name_type` is not defined in metadata and it will not appear on the datacite rendered page. Just checked it again on my local machine.
An example (`name_type` is not defined for my name but for Joe Doe)
```xml
<creators>
  <creator>
    <creatorName nameType="Organizational">Leibniz Institute for Astrophysics Potsdam</creatorName>
  </creator>
  <creator>
    <creatorName>Makan, Kirill</creatorName>
    <givenName>Kirill</givenName>
    <familyName>Makan</familyName>
    <nameIdentifier nameIdentifierScheme="ORCID" schemeURI="http://orcid.org/">0000-0000-0000-0000</nameIdentifier>
    <affiliation affiliationIdentifier="https://ror.org/03mrbr458" affiliationIdentifierScheme="ROR" schemeURI="http://ror.org">Leibniz Institute for Astrophysics Potsdam</affiliation>
  </creator>
  <creator>
    <creatorName nameType="Personal">Doe, Joe</creatorName>
    <givenName>Joe</givenName>
    <familyName>Doe</familyName>
    <nameIdentifier nameIdentifierScheme="ORCID" schemeURI="http://orcid.org/">0000-0000-0000-0000</nameIdentifier>
  </creator>
  </creators>
```